Pages/wishlist.html

```diff
@@ -0,0 +1,189 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <title>Wishlist | Plantly</title>
+
+    <!-- Font -->
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@100;300;400;500;700;900&display=swap"
+      rel="stylesheet"
+    />
+
+    <!-- Stylesheets -->
+    <link rel="stylesheet" href="https://aceui.netlify.app/aceui-styles.css" />
+    <link rel="stylesheet" href="../Styles/productListing.css" />
+    <link rel="stylesheet" href="../style.css" />
+
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"
+    />
+  </head>
+
+  <body>
+    <!-- Main Header -->
+    <header class="main-header flex-row px-8">
+      <nav class="nav-bar">
+        <a href="https://plantlybyaish.netlify.app">
```

Comment:
`a` tag can be replaced with `Link` from `react-router-dom`. This will prevent unnecessary page refreshes and improve user experience

Reply:
This PR is actually for E-commerce screens, will update it when I'll start with React app.

Reply:
Oh okay

Reply:
My bad